src/pages/updates/work-in-progress/index.mdx

```diff
@@ -0,0 +1,79 @@
+---
+title: Work in progress
+description:
+  This page provides early visibility for new Carbon concepts and current work
+  in progress.
+---
+
+<PageDescription>
+
+An opportunity to see early Carbon concepts and ideas, and experiment with work
+in progress.
+
+</PageDescription>
+
+<InlineNotification>
+  This work in progress is not actively supported by the Carbon team, and is not
+  approved for production use. We gratefully accept feedback or comment.
+</InlineNotification>
+
+## Introduction
+
+The Carbon team is always exploring different ways to serve our users. We work
+in the open, and this page is here to give you early previews of what we're
+working on.
+
+That could mean making an unfinished Sketch kit available for download, linking
+to GitHub issues for proposed components and patterns, or presenting graphics
+files of concepts on how we might present content.
+
+Not everything here may make it into production, other pieces might become
+central pieces of the design system. It's an early opportunity for you to
+provide feedback and influence Carbon's direction.
+
+### Tools and resources
+
+<CardGroup>
+  <MiniCard onClick={() => fathom('trackGoal', 'GC8C8OMU', 0)} title="Carbon mobile sketch kit" href="/">
+
+
+![Sketch icon](/images/sketch-icon.png)
```

Comment:
```suggestion
<MdxIcon name="sketch" />
```

This will pull in the correct icon so its consistent (and with transparent background) across the site

Reply:
This doesn't work for the Sketch icon (still get white background). For the github icon, it does give me the correct transparent background, but the icon is oversized (sizing for full size Card icon size, rather than MiniCard).

Reply:
main image might need to get updated, and the icon component or minicard. bigger than this PR then, you can ignore me 😉

Reply:
👍 OK. I'm going to revert to just using the default config icons for the first draft, and then we can address the icon work after that.

Reply:
Super weird, i pulled down your PR and i'm totally seeing the white background, but the GTC site itself is displaying the icons just fine. Trying to find the mismatch.

Reply:
Huh. OK, I'll leave making changes to something that's in the core Carbon site styling right now. But, @alisonjoseph...something that's safe to remove? Separate issue to change.

Reply:
Created a bug issue specific to that - https://github.com/carbon-design-system/carbon-website/issues/1530